docs/spec/v1beta2/kustomizations.md

```diff
@@ -1260,32 +1260,10 @@ env:
 
 #### Azure Key Vault
 
-While making use of [AAD Pod Identity](https://github.com/Azure/aad-pod-identity),
```

Comment:
Please undo this change, v1beta2 is a deprecated API and its code and docs are considered readonly.